OrbitQt/orbitlivefunctions.ui

```diff
@@ -32,19 +32,103 @@
    <property name="bottomMargin">
     <number>0</number>
    </property>
-   <item row="0" column="0">
-    <widget class="OrbitDataViewPanel" name="data_view_panel_" native="true"/>
-   </item>
-   <item row="1" column="0">
-    <layout class="QVBoxLayout" name="iteratorLayout">
-     <item>
-      <widget class="QLabel" name="instructionsLabel">
-       <property name="text">
-        <string>Select functions above, right-click, and select 'Add iterator(s)' to add iterators for instrumented functions.</string>
-       </property>
-      </widget>
-     </item>
-    </layout>
+   <item>
+    <widget class="QSplitter" name="splitter">
+     <property name="orientation">
+      <enum>Qt::Vertical</enum>
+     </property>
+     <widget class="OrbitDataViewPanel" name="data_view_panel_" native="true">
+      <property name="sizePolicy">
+       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
+        <horstretch>0</horstretch>
+        <verstretch>0</verstretch>
+       </sizepolicy>
+      </property>
+     </widget>
+     <widget class="QWidget" name="verticalLayoutWidget">
+      <layout class="QVBoxLayout" name="verticalLayout_3">
+       <item>
+        <widget class="QLabel" name="instructionsLabel">
+         <property name="styleSheet">
+          <string notr="true">margin-top: 5px;</string>
+         </property>
+         <property name="text">
+          <string>Select functions above, right-click, and select 'Add iterator(s)' to add iterators for instrumented functions.</string>
+         </property>
+         <property name="margin">
+          <number>0</number>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QScrollArea" name="iteratorScrollArea">
+         <property name="autoFillBackground">
+          <bool>true</bool>
+         </property>
+         <property name="styleSheet">
+          <string notr="true">QScrollArea {
+	border: 0;
+}</string>
```

Comment:
Fix indentation here?

Reply:
QT Designer saves it like this and it will just re-appear if changes are made, so I think we should just leave it, even though it looks weird.